backend/src/models/db.js

```diff
@@ -0,0 +1,24 @@
+const { Client } = require("pg");
+
+const DB_DATA = {
+  user: process.env.PGUSER || 'postgres',
+  host: process.env.PGHOST || 'docker.vm',
+  database: process.env.PGDB || 'postgres',
+  password: process.env.PGPASSWORD || '123456',
+  port: process.env.PGPORT || 5432
+};
+
+module.exports = {
+  query: async (text, params) => {
+    const client = new Client(DB_DATA);
+    await client.connect();
+    return client.query(text, params).then(res => {
```

Comment:
I think we need a `catch` block here, and we can put `client.end()` into the `finally` block

Reply:
At the point when `.then` is run all data was already fetched from db, therefore there is no need for the client and we can release it immediately. All queries exported this way are disposable, they don't keep any resources more than it is necessary.

Reply:
But you are right, finally is proper way.